From 2293982a73791d1e6c1ae0d8ab80b54a2853e17d Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Mon, 14 Aug 2017 11:27:20 +1000
Subject: [PATCH 3/5] rc: Add lrelease override for qmake4 macro

Came across this when fixing x2goclient. It's not commonly used, but will fix
building on some packages.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 ypkg2/rc.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ypkg2/rc.yml b/ypkg2/rc.yml
index 9d36ba6..761e39e 100644
--- a/ypkg2/rc.yml
+++ b/ypkg2/rc.yml
@@ -24,7 +24,7 @@ actions:
     - qmake: |
         qmake QMAKE_CFLAGS_RELEASE="%CFLAGS%" QMAKE_CXXFLAGS_RELEASE="%CXXFLAGS%" QMAKE_LFLAGS="%LDFLAGS%"
     - qmake4: |
-        qmake-qt4 QMAKE_CFLAGS_RELEASE="%CFLAGS%" QMAKE_CXXFLAGS_RELEASE="%CXXFLAGS%" QMAKE_LFLAGS="%LDFLAGS%" QMAKE_MOC=/usr/bin/moc-qt4 QMAKE_RCC=/usr/bin/rcc-qt4 QMAKE_UIC=/usr/bin/uic-qt4
+        qmake-qt4 QMAKE_CFLAGS_RELEASE="%CFLAGS%" QMAKE_CXXFLAGS_RELEASE="%CXXFLAGS%" QMAKE_LFLAGS="%LDFLAGS%" QMAKE_LRELEASE=/usr/bin/lrelease-qt4 QMAKE_MOC=/usr/bin/moc-qt4 QMAKE_RCC=/usr/bin/rcc-qt4 QMAKE_UIC=/usr/bin/uic-qt4
     - patch: |
         patch -t -E --no-backup-if-mismatch -f
     # Only works if the user has a series file. They can provide the name to
-- 
2.14.2

